src/components/common/__tests__/TreeExplorerTreeNode.spec.ts

```diff
@@ -26,7 +26,8 @@ describe('TreeExplorerTreeNode', () => {
     totalLeaves: 3,
     icon: 'pi pi-folder',
     type: 'folder',
-    handleRename: () => {}
+    handleRename: () => {},
+    badgeText: '3'
```

Comment:
Why this is change is necessary? If we don't specify badgeText it should it should fallback to totalLeaves

Reply:
this is test code that is emitting a `{}` instead of the real object, ie all variables must be explicitly defined (it should probably not be a `{}` block but that's a separate issue lol)